tox.ini

```diff
@@ -1,14 +1,54 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/buildout-recipe
 [tox]
-envlist = flake8,py27,py35,py36,py37,py38,pypy
+minversion = 3.18
+envlist =
+    lint
+    py27
+    py35
+    py36
+    py37
+    py38
+    py39
+    pypy
+    pypy3
+    coverage
 
 [testenv]
-commands =
-    zope-testrunner --test-path=src []
+usedevelop = true
 deps =
-    .[test]
+    zope.testrunner
+commands =
+    zope-testrunner --test-path=src {posargs:-vc}
+extras =
+    test
 
-[testenv:flake8]
-basepython = python3.6
+[testenv:lint]
+basepython = python3
 skip_install = true
-deps = flake8
-commands = flake8 --doctests src setup.py
+deps =
+    flake8
+    check-manifest
+    check-python-versions
+commands =
+    flake8 src setup.py
+    check-manifest
+    check-python-versions
+
+[testenv:coverage]
+basepython = python3
+allowlist_externals =
+    mkdir
+setenv =
+    COVERAGE_PROCESS_START={toxinidir}/.coveragerc
+deps =
+    coverage
+    coverage-python-version
+    zope.testrunner
+commands =
+    mkdir -p {toxinidir}/parts/htmlcov
+    coverage erase
+    coverage run -m zope.testrunner --test-path=src []
+    coverage combine
```

Comment:
Does the `coverage combine` actually provide some additional value here?

Reply:
The tests are run in separate processes by buildout. `combine` is needed to collect the coverage of the separate processes.